src/__tests__/to-have-style.js

```diff
@@ -214,7 +214,7 @@ describe('.toHaveStyle', () => {
         <span data-testid="color-example" style="font-size: 12rem">Hello World</span>
       `)
       expect(() => {
-        expect(queryByTestId('color-example')).toHaveStyle({ fontSize: '12px' })
+        expect(queryByTestId('color-example')).toHaveStyle({fontSize: '12px'})
```

Comment:
That one came from the husky pre-commit hook formatting the whole file. By the way running `npm run format` would have changed a bunch of other files, too.